src/index.js

```diff
@@ -1,13 +0,0 @@
-import './index.css';
-import { invAPI , getPokemon } from "./modules/api.js";
-
-const list = document.querySelector(".pokemon-data");
-const test = document.querySelector(".test");
-
-for (let i = 1; i <= 10; i+= 1){
-    const pokemon = await getPokemon(i);
-    console.log(pokemon.name);
-    // list.innerHTML += `<li class="score">${pokemon.name}</li>`
-}
-
-document.addEventListener('DOMContentLoaded', getPokemon());
```

Comment:
Though you may not use this, do not delete this part of the code, as it is what allows Webpack to print out the website with the correct styling and display